lib/Drupal/Generic/Drupal7.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Generic\Drupal7.
+ */
+
+namespace Drupal\Generic;
+
+class Drupal7 {
+  public function __call($method, $args) {
+    if (function_exists('drupal_' . $method)) {
```

Comment:
Ok I'll change it now.

 -*φol d.*-

On Thu, Jun 4, 2015 at 8:53 AM, Fabian Franz <notifications@github.com>
wrote:

> In lib/Drupal/Generic/Drupal7.php
> <https://github.com/LionsAd/service_container/pull/24#discussion_r31696936>
> :
>
> > @@ -0,0 +1,17 @@
> > +<?php
> > +
> > +/**
> > + * @file
> > + * Contains \Drupal\Generic\Drupal7.
> > + */
> > +
> > +namespace Drupal\Generic;
> > +
> > +class Drupal7 {
> > +  public function __call($method, $args) {
> > +    if (function_exists('drupal_' . $method)) {
>
> Not too keen on that specialization.
>
> I think
>
> $this->drupal7->drupal_get_filename()
>
> is better than
>
> $this->drupal7->get_filename()
>
> where you don't know which function it will call ...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/LionsAd/service_container/pull/24/files#r31696936>.
>